rinkeby_migrations/15_set_new_MonetarySupervisor_live.js

```diff
@@ -0,0 +1,86 @@
+/* set new MonetarySupervisor deployed in prev step. live and adjust new LTD params
+    It's switching it live, i.e.:
+        - granting permissions for new MS in deployed contracts and revoking from old MS contract
+        - setting new MS in Locker and LoanManager contracts
+        - migrating totalLoan and totalLockedAmount from old MS
+*/
+const Migrations = artifacts.require("./Migrations.sol");
+const MonetarySupervisor = artifacts.require("./MonetarySupervisor.sol");
+const TokenAEur = artifacts.require("./TokenAEur.sol");
+const InterestEarnedAccount = artifacts.require("./InterestEarnedAccount.sol");
+const AugmintReserves = artifacts.require("./AugmintReserves.sol");
+const LoanManager = artifacts.require("./LoanManager.sol");
+const Locker = artifacts.require("./Locker.sol");
+const FeeAccount = artifacts.require("./FeeAccount");
+
+module.exports = function(deployer, network) {
+    deployer.then(async () => {
+        // NB: web3.version.network throws error with truffle migrate --dry-run...
+        if (network === "rinkeby" || network === "rinkeby-fork") {
+            // Truffle artifacts are in unknown state when truffle migrate starts from this step on rinkeby.
+            // Therefore we can't use addresses from there.
+            // But this script will be run only ONCE on rinkeby so it's fine to hardcode addresses
+            const ratesAddress = "0xcA8100FCcb479516A5b30f8Bc5dAeA09Fb7a7473";
```

Comment:
Could you get these addresses from an Abiniser environment json?

Reply:
We could get the latest deployed address for a given abi. I would feel the script a bit more flaky and difficult to check before running compared to hardcoding address where we can check the addresses for a given deployment's on the under-the-hood page.